tests/lac/vector_type_traits_is_serial_11.output

```diff
@@ -0,0 +1,3 @@
+
+DEAL::0
```

Comment:
Since this test is for `PETSc` and with MPI types, the output file name must be adjusted to `vector_type_traits_is_serial_11.with_petsc=true.with_mpi=true.output`